core/src/main/scala/org/apache/spark/deploy/history/HistoryServerArguments.scala

```diff
@@ -61,6 +61,7 @@ private[history] class HistoryServerArguments(conf: SparkConf, args: Array[Strin
       |Usage: HistoryServer [options]
       |
       |Options:
+      |  --dir DIR (-d DIR)          Sets spark.history.fs.logDirectory,deprecated use conf instead
```

Comment:
",deprecated" looks funny. Maybe write "Deprecated; set spark.history.fs.logDirectory directly" on the next line?